v22.1/demo-fault-tolerance-and-recovery.md

```diff
@@ -291,48 +283,41 @@ To be able to tolerate 2 of 5 nodes failing simultaneously without any service i
     --background
     ~~~
 
-2. Use the [`ALTER RANGE ... CONFIGURE ZONE`](configure-zone.html) command to change the cluster's `default` replication factor to 5:
+1. Use the [`ALTER RANGE ... CONFIGURE ZONE`](configure-zone.html) command to change the cluster's `default` replication factor to 5:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    $ cockroach sql --execute="ALTER RANGE default CONFIGURE ZONE USING num_replicas=5;" --insecure --host=localhost:26000
+    $ cockroach sql --insecure --host=localhost:26000 \
+    --execute="ALTER RANGE default CONFIGURE ZONE USING num_replicas=5;"
     ~~~
 
-3. Back in the DB Console **Overview** dashboard, watch the replica count increases and even out across all 6 nodes:
+1. In the DB Console **Overview** dashboard, watch the replica count increase and even out across all 6 nodes:
 
-    <img src="{{ 'images/v22.1/fault-tolerance-8.png' | relative_url }}" alt="DB Console" style="border:1px solid #eee;max-width:100%" />
+    <img src="{{ 'images/v22.1/fault-tolerance-8.png' | relative_url }}" alt="DB Console Cluster Restore" style="border:1px solid #eee;max-width:100%" />
 
     This shows the cluster up-replicating so that each range has 5 replicas, one on each node.
 
 ## Step 9. Simulate two simultaneous node failures
 
-Gracefully shut down two nodes, specifying the [process IDs you retrieved earlier](#step-5-simulate-a-single-node-failure):
-
-{% include_cached copy-clipboard.html %}
-~~~ shell
-kill -TERM 4510
-~~~
+Gracefully shut down **2 nodes**, specifying the [process IDs you retrieved earlier](#step-5-simulate-a-single-node-failure):
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-kill -TERM 4622
+kill -TERM {process-id}
 ~~~
 
-## Step 10. Check load continuity and cluster health
-
-1. Like before, go to the DB Console, click **Metrics** on the left, and verify that the cluster as a whole continues serving data, despite 2 nodes being offline:
+## Step 10. Check cluster status and service continuity
 
-    <img src="{{ 'images/v22.1/fault-tolerance-9.png' | relative_url }}" alt="DB Console" style="border:1px solid #eee;max-width:100%" />
+1. Click **Overview** on the left, and verify the state of the cluster:
 
-    This shows that when all ranges are replicated 5 times, the cluster can tolerate 2 simultaneous node outages because the surviving nodes have a majority of each range's replicas (3/5).
+    <img src="{{ 'images/v22.1/fault-tolerance-9.png' | relative_url }}" alt="DB Console Cluster Health" style="border:1px solid #eee;max-width:100%" />
 
-2. To verify this further, use the `cockroach sql` command to count the number of rows in the `ycsb.usertable` table and verify that it is still serving reads:
+1. To verify that the cluster still serves data, use the `cockroach sql` command to interact with the cluster.
 
+    Count the number of rows in the `ycsb.usertable` table to see that it serves reads:
```

Comment:
Again, not sure if this needs to be something like `ycsb.public.usertable`. I'm still naive to how the table names are displayed, so apologies if this is irrelevant!